tools/zetasql_helper/zetasql_helper/local_service/local_service.proto

```diff
@@ -11,6 +28,9 @@ service ZetaSqlHelperLocalService {
   rpc Hello(HelloRequest) returns (HelloResponse) {
   }
 
+  rpc Tokenize(TokenizeRequest) returns (TokenizeResponse) {
+  }
+
 }
 
 message HelloRequest {
```

Comment:
we don't need this HelloRequest/Response right?

Reply:
No we don't need it. I will remove it.